tests/variables2/basic_crud.test.js

```diff
@@ -46,9 +46,10 @@ describe('Test variable get delete list calls', () => {
 		const res = await request(config.getDirektivHost())
 			.get(`/api/v2/namespaces/${ namespace }/variables`)
 		expect(res.statusCode).toEqual(200)
-		expect(res.body).toMatchObject({
```

Comment:
@alankm , Why this change is necessary?

Reply:
This test started failing for me. Not every time. I think it's random. Give it a try a bunch of times on main to see for yourself maybe? I think this happened because you didn't use the classic `expectArrayContaining` (or whatever it's called) so it ended up needing to be an exact match or something. Maybe the returned order is not 100% deterministic?

Reply:
Well, expectArrayContaining is not always good. It depends what you are testing. I will check it out later. Feel free to merge.